var/spack/repos/builtin/packages/cudnn/package.py

```diff
@@ -5,229 +5,172 @@
 
 import os
 from spack import *
-
-
-class Cudnn(Package):
-    """NVIDIA cuDNN is a GPU-accelerated library of primitives for deep
-    neural networks"""
-
-    homepage = "https://developer.nvidia.com/cudnn"
-
-    # Latest versions available at:
-    #     https://developer.nvidia.com/rdp/cudnn-download
-    # Archived versions available at:
-    #     https://developer.nvidia.com/rdp/cudnn-archive
-    # Note that download links don't work from command line,
-    # need to use modified URLs like in url_for_version.
-
-    maintainers = ['adamjstewart']
-
-    # cudNN 8.0.4
-    version('8.0.4.30-11.1-linux-x64',
-            sha256='8f4c662343afce5998ce963500fe3bb167e9a508c1a1a949d821a4b80fa9beab')
-    version('8.0.4.30-11.1-linux-ppc64le',
-            sha256='b4ddb51610cbae806017616698635a9914c3e1eb14259f3a39ee5c84e7106712')
-    version('8.0.4.30-11.0-linux-x64',
-            sha256='38a81a28952e314e21577432b0bab68357ef9de7f6c8858f721f78df9ee60c35',
-            preferred=True)
-    version('8.0.4.30-11.0-linux-ppc64le',
-            sha256='8da8ed689b1a348182ddd3f59b6758a502e11dc6708c33f96e3b4a40e033d2e1')
-    version('8.0.4.30-10.2-linux-x64',
-            sha256='c12c69eb16698eacac40aa46b9ce399d4cd86efb6ff0c105142f8a28fcfb980e')
-    version('8.0.4.30-10.2-linux-ppc64le',
-            sha256='32a5b92f9e1ef2be90e10f220c4ab144ca59d215eb6a386e93597f447aa6507e')
-    version('8.0.4.30-10.1-linux-x64',
-            sha256='eb4b888e61715168f57a0a0a21c281ada6856b728e5112618ed15f8637487715')
-    version('8.0.4.30-10.1-linux-ppc64le',
-            sha256='690811bbf04adef635f4a6f480575fc2a558c4a2c98c85c7090a3a8c60dacea9')
+import platform
+
+_versions = {
+    # cuDNN 8.0.4
+    '8.0.4.30-11.1': {
+        'Linux-x86_64': ('8f4c662343afce5998ce963500fe3bb167e9a508c1a1a949d821a4b80fa9beab'),
```

Comment:
You might want to get rid of the parentheses around these, they aren't doing anything.

Reply:
Done